user_modules/shared_testcases/testCasesForExternalUCS.lua

```diff
@@ -0,0 +1,179 @@
+---------------------------------------------------------------------------------------------
+-- Common functions for External_UCS test scripts
+---------------------------------------------------------------------------------------------
+local mobile_session = require("mobile_session")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local json = require("modules/json")
+local sdl = require('SDL')
+
+local utils = { }
+
+-- [[ Variables ]]
+	utils.HMIAppIds = { }
+	utils.pts = nil
+
+-- [[ Functions ]]
+	function utils.createTableFromJsonFile(file)
+	  local f = io.open(file, "r")
+	  local content = f:read("*all")
+	  f:close()
+	  return json.decode(content)
+	end
+
+	function utils.createJsonFileFromTable(table, file)
+	  local f = io.open(file, "w")
+	  f:write(json.encode(table))
+	  f:close()
+	end
+
+	function utils.checkSDLStatus(test, expStatus)
+	  local actStatus = sdl:CheckStatusSDL()
+	  print("SDL status: " .. tostring(actStatus))
+	  if actStatus ~= expStatus then
+	    local msg = "Expected SDL status: " .. expStatus .. ", actual: " .. actStatus
+	    test:FailTestCase(msg)
+	  end
+	end
+
+	function utils.removeLPT()
+		local folderPath = commonPreconditions:GetPathToSDL() ..
+      commonFunctions:read_parameter_from_smart_device_link_ini("AppStorageFolder")
+    os.execute("rm -rf " .. folderPath)
+	end
+
+	function utils.removePTS()
+    local filePath = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath") ..
```

Comment:
@istoimenova, Thank you for suggestion, but using 'rm -rf' command will remove files (or folders) silently despite of the file (or folder) existence